# Goal definition
geometry_msgs/PoseStamped target_pose
float64 center_x # x component of center of the circle
float64 center_y # y component of center of the circle
float64 center_z # z component of center of the circle
string tcp_link # e.g., "ur5_gripper_tcp", "ur5_suction_tcp" (optional, defaults to "ur5_gripper_tcp")
float64 velocity_scaling # velocity scaling factor (0.0 to 1.0, defaults to 0.1)
float64 acceleration_scaling # acceleration scaling factor (0.0 to 1.0, defaults to 0.1)
---
# Result definition
bool success
string message
---
# Feedback definition
geometry_msgs/PoseStamped current_pose
